templates/release-docs-changelog.yml

```
@@ -0,0 +1,78 @@
+spec:
+  inputs:
+    stage:
+      description: The pipeline stage where to perform the docs release
+      default: publish
+    runner:
+      description: GitLab CI runner tag
+      default: hetzner-amd-beefy
+    alpine-git-tag:
+      description: Tag for alpine/git container image
+      default: latest
+    remote_changelog_file:
+      description: Remote changelog file to be used
+---
+
+
+release:mender-docs-changelog:
+  stage: $[[ inputs.stage ]]
+  tags:
+    - $[[ inputs.runner ]]
+  needs: []
+  image:
+    name: alpine/git:$[[ inputs.alpine-git-tag ]]
+    entrypoint: [""]
+  rules:
+    # Only make available for stable tags
+    - if: '$CI_COMMIT_TAG =~ /^v?\d+\.\d+\.\d+$/'
+      allow_failure: true
+  variables:
+    GITHUB_CHANGELOG_REPO_URL: "mender-docs-changelog"
+  before_script:
+    # Check required input parameters
+    - |
+      if [ -z "$[[ inputs.remote_changelog_file ]]" ]; then
+        echo "ERROR: Missing template argument 'remote_changelog_file'"
+        echo "Please specify the remote changelog name in your include section:"
+        echo "  inputs:"
+        echo "    remote_changelog_file: PATH_TO_CHANGELOG_FILE"
+        exit 1
+      fi
+    # Check required environment variables
+    - |
+      if [ -z "$GITHUB_USER_EMAIL" ] || [ -z "$GITHUB_USER_NAME" ] || [ -z "$GITHUB_CLI_TOKEN" ] || [ -z "$GITHUB_BOT_TOKEN_REPO_FULL" ]; then
+        echo "ERROR: Missing required environment variables:"
+        echo "  - GITHUB_USER_EMAIL: ${GITHUB_USER_EMAIL:-'NOT SET'}"
+        echo "  - GITHUB_USER_NAME: ${GITHUB_USER_NAME:-'NOT SET'}"
+        echo "  - GITHUB_CLI_TOKEN: $([ -z "$GITHUB_CLI_TOKEN" ] && echo 'NOT SET' || echo 'SET')"
+        echo "  - GITHUB_BOT_TOKEN_REPO_FULL: $([ -z "$GITHUB_BOT_TOKEN_REPO_FULL" ] && echo 'NOT SET' || echo 'SET')"
+        exit 1
+      fi
+    # Install required tools
+    - apk add --no-cache python3 curl github-cli
+    - curl -fsSL https://cli.github.com/packages/alpine/install.sh | sh
+    # Setting up git
+    - git config --global user.email "${GITHUB_USER_EMAIL}"
+    - git config --global user.name "${GITHUB_USER_NAME}"
+    # GITHUB_TOKEN for Github cli authentication
+    - export GITHUB_TOKEN=${GITHUB_CLI_TOKEN}
+  script:
+    - git clone https://${GITHUB_USER_NAME}:${GITHUB_BOT_TOKEN_REPO_FULL}@github.com/mendersoftware/${GITHUB_CHANGELOG_REPO_URL}
+    - cd ${GITHUB_CHANGELOG_REPO_URL}
+    - if [ ! -e $[[ inputs.remote_changelog_file ]] ]; then
+        echo "ERROR: Remote changelog file $[[ inputs.remote_changelog_file ]] must exists"
+        exit 1
+      fi
```

Incorrect yaml syntax

```suggestion
    - |
      if [ ! -e $[[ inputs.remote_changelog_file ]] ]; then
        echo "ERROR: Remote changelog file $[[ inputs.remote_changelog_file ]] must exists"
        exit 1
      fi
```